.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,32 @@
+name: shellcheck
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+permissions: {}
+
+jobs:
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-24.04
+    env:
+      VERSION: v0.10.0
```

Comment:
Can you ensure renovate updates this?

Reply:
I can open an issue for it, I don't know how to do that.

Reply:
https://github.com/open-telemetry/opentelemetry-operator/issues/3676 is open